site/kube-addon.md

```diff
@@ -144,3 +144,7 @@ The list of variables you can set is:
 * WEAVE\_EXPOSE\_IP - set the IP address used as a gateway from the
   Weave network to the host network - this is useful if you are
   configuring the addon as a static pod.
+* WEAVE\_MTU - Weave Net defaults to 1410 bytes, but you can set a
+  smaller size if your underlying network has a tighter limit, or set
+  a larger size for better performance if your network supports jumbo
+  frames.
```

Comment:
Sub-sections can be linked to if an anchor is created, e.g. like https://www.weave.works/docs/net/latest/using-weave/service-management/#routing